### PR TITLE
kakaopage fix

### PR DESCRIPTION
**Pull request by x4nk1t**

### CODE REVIEW

File: src/web/mjs/connectors/kakaopage.mjs [hunk 1]
```diff
@@ -19,6 +19,7 @@ export default class kakaopage extends Connector {
         let id = uri.searchParams.get('seriesId');
```
Comment (27136773) [inline src/web/mjs/connectors/kakaopage.mjs]: May use short circuiting
```javascript
let id = uri.searchParams.get('seriesId') || uri.pathname.split('/').filter(part => part).pop();
```

**Review by 27136773 — CHANGES_REQUESTED** [review]
Fix lint errors before merge.
Optional: Minor improvements possible.

**Review by 27136773 — APPROVED** [review]
Looks good

**Comment by x4nk1t** [comment]
Closes #3208

**Pull request merged**